pages/_app.tsx

```diff
@@ -1,6 +1,7 @@
 import { SessionProvider } from "next-auth/react"
 
 import Layout from '../components/Layout'
+import '../style.css'
```

Comment:
If you add the global file instead of the one you made, you don't need to import this file.